app/scripts/views/AppView.js

```diff
@@ -62,6 +63,9 @@ define([  'backbone',
         //we will never get to the render event, because model is already initialized and therefore it
         //won't generate 'change' event
         this.render();
+
+        // ThreeViewer does not expose any public method therefore just call init
+        new ThreeViewer({app: this.model});
```

Comment:
Fine for now.  In the future, you can imagine specifying which DOM element to inject into.